app/views/pane-editorial-comments.html

```diff
@@ -0,0 +1,102 @@
+<section
+    ng-controller="EditorialCommentsCtrl"
+    class="tabcontent comments-panel">
+
+  <div class="panel-title">
+    {{ ::('aes.label.editorialcomments'|trans) }}
+    <div class="load-anim" style="
+    float: right; margin: 0px;
+" ng-hide="!isLoading">
+      <div class="rect1"></div>
+      <div class="rect2"></div>
+      <div class="rect3"></div>
+      <div class="rect4"></div>
+      <div class="rect5"></div>
+    </div>
+  </div>
+    <div
+    class="add-element"
+    ng-class="{expanded: adding}"
+    >
+    <div class="heading">
+      <button
+         class="btn btn-sf-invisible"
+         data-test-button="add"
+         ng-click="adding = true"
+         ng-hide="adding"
+         >
+        <span class="glyphicon glyphicon-plus"></span><span class="add-text">{{ ::('aes.label.postcomment'|trans) }}</span>
+      </button>
+      <button
+         class="btn btn-sf-invisible"
+         ng-click="adding = false"
+         ng-show="adding"
+         >
+        <span class="glyphicon glyphicon-minus"></span><span class="add-text">{{ ::('aes.label.postcomment'|trans) }}</span>
+      </button>
+    </div>
+    <form name="addEditorialComment"
+       ng-show="adding"
+       class="body"
+       >
+      <div class="input-name">{{ ::('Comment'|trans) }}</div>
+      <textarea
+         class="editor-input"
+         ng-model="create.comment"
+         ng-disabled="isSending"
+         required
+         ></textarea>
+      <div class="controls">
+        <button
+           class="btn btn-primary btn-sm pull-right"
+           data-test-button="new"
+           ng-click="add({editorial_comment:create})"
+           ng-disabled="isSending || addEditorialComment.$invalid"
+           >
+          {{ ::('Send'|trans) }}
+        </button>
+        <button
+           class="btn btn-default btn-sm pull-right"
+           ng-click="cancel()"
+           ng-disabled="isSending"
+           >
+          {{ ::('Cancel'|trans) }}
+        </button>
+      </div>
+    </form>
+  </div>
+
+    <div class="list nested" ng-class="{'editorial-low': adding, 'editorial-high': !adding}">
+    <div class="liner" ng-init="fetchComments()">
+      <editorial-comment
+          model="comment"
+          ng-repeat="comment in filtered = (comments.displayed | sortComments:'Nested')"
```

Comment:
Consider adding `track by` to avoid unnecessary DOM node re-creation, especially if the list of comments tends to change.